src/main/asciidoc/micrometer/influx/main.adoc

```diff
@@ -0,0 +1,122 @@
+
+image::scdf-micrometer-influxdb-grafana-architecture.png[scaledwidth="100%", title="SCDF metrics analyzis with InfluxDB and Grafana"]
+
+Out of the box, SCSt sends https://docs.spring.io/spring-boot/docs/2.0.1.RELEASE/reference/htmlsingle/#production-ready-metrics-meter[core metrics] such as `CPU`, `Memory`, `MVC` and `Health` to name some. Amongh those the https://docs.spring.io/spring-integration/docs/current/reference/html/system-management-chapter.html#micrometer-integration[Spring Integration metrics] allows computing the `Rate` and the `Latency` of the messages in the SCDF streams.
+
+NOTE: Unlike Spring Cloud Data Flow Metrics Collector, metrics here are send synchronously over HTTP not through a Binder channel topic.
+
+All Spring Cloud Stream App Starers enrich the standard  http://micrometer.io/docs/concepts#_supported_monitoring_systems[dimensional tags] with the following SCDF specific tags:
+[width="100%",options="header"]
+|====================
+| tag name | SCDF property | default value
+| stream.name |  spring.cloud.dataflow.stream.name | unknown
+| application.name | spring.cloud.dataflow.stream.app.label | unknown  
+| instance.index | instance.index | 0 
+| application.guid | spring.cloud.application.guid | unknown 
+| application.type | spring.cloud.dataflow.stream.app.type | unknown
+|====================
+
+NOTE: For custom app starters that don't extend from the https://github.com/spring-cloud-stream-app-starters/core[core] parent, you should add the `app-starters-common` : `org.springframework.cloud.stream.app` dependecny to enable the SCDF tags.
+
+Below we will present the steps to prep, configure the demo of Spring Cloud Data Flow's `Local` server integration with `InfluxDB`. For other deployment environment, such as `Cloud Foundry` or `Kubernetes`, addition configurations might be required.
+
+==== Prerequisites
+
+* A Running Data Flow Shell
+include::{docs_dir}/shell.adoc[]
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Spring Cloud Stream 2.x based https://github.com/spring-cloud-stream-app-starters/time/blob/master/spring-cloud-starter-stream-source-time/README.adoc[Time] and  https://github.com/spring-cloud-stream-app-starters/log/blob/master/spring-cloud-starter-stream-sink-log/README.adoc[Log] appcliations starters, pre-build with `io.micrometer:micrometer-registry-influx` dependency. 
++
+NOTE: Next versions of the https://start-scs.cfapps.io/[SCSt App Initializr] utility would add support for Micrometer dependecies to facilitate the injection of micrometer-registries with SCSt apps. 
+
+==== Building and Running the Demo
+
+. Register `time` and `log` applications pre-build `io.micrometer:micrometer-registry-influx`. Coming to next version of https://start-scs.cfapps.io/[SCSt App Initializr] is support adding Micrometer registry dependencies as well.
```

Comment:
Can we modify/re-phrase the statements in this line?

Maybe with the following:
1) pre-build -> that are pre-built with
2) The next version of ... supports adding ..